app/src/pages/experiment/ExperimentCompareTable.tsx

```diff
@@ -456,82 +456,82 @@ export function ExperimentCompareTable(props: ExampleCompareTableProps) {
 
   const isEmpty = rows.length === 0;
 
-  // Make sure the table is at least 1280px wide
-
   return (
-    <Flex direction="column" height="100%">
-      <View
-        paddingTop="size-100"
-        paddingBottom="size-100"
-        paddingStart="size-200"
-        paddingEnd="size-200"
-        borderBottomColor="grey-300"
-        borderBottomWidth="thin"
-        flex="none"
-      >
-        <ExperimentRunFilterConditionField
-          onValidCondition={setFilterCondition}
-        />
-      </View>
-      <div css={tableWrapCSS}>
-        <table
-          css={css(tableCSS, borderedTableCSS)}
-          style={{
-            ...columnSizeVars,
-            width: table.getTotalSize(),
-            minWidth: "100%",
-          }}
+    <View overflow="auto">
```

Comment:
strange that the sizing in the table body itself doesn't work here. I might not understand this TBH. Stamping to unblock.

Reply:
Something to do with the filter bar above and the sticky table headers. Not sure the exact root cause to be honest